File: Arcade/TheCore/Java/MakeArrayConsecutive2.java
final class MakeArrayConsecutive2 {
    
    int makeArrayConsecutive2(int[] statues) {
        Arrays.sort(statues);
        
        final int max = statues[statues.length - 1];
        final int min = statues[0];
        int additionalStatues = 0;
        
        for (int statue = min; statue <= max; statue++)
            if (Arrays.binarySearch(statues, statue) < 0)
                additionalStatues++;
        
        return additionalStatues;
    }
}

File: Arcade/TheCore/Javascript/makeArrayConsecutive2.js
function makeArrayConsecutive2(statues) {
    const min = Math.min.apply(null, statues)
    const max = Math.max.apply(null, statues)
    
    let additionalStatues = 0
    
    for (let statue = min; statue <= max; statue++)
        if (!statues.includes(statue)) additionalStatues++
        
    return additionalStatues
}